helm/values.yaml

```diff
@@ -155,3 +155,20 @@ servicemonitor:
   interval: 15s
   secure: false
   tlsConfig: {}
+
+compatibilityJob:
+  enabled: false
+  name: schema-registry-compatibility-job
+  hook: post-install, pre-upgrade
+  hookDeletePolicy: before-hook-creation
+  backoffLimit: 3
+  ttlSecondsAfterFinished: 600
+  activeDeadlineSeconds: 300
+  image: curlimages/curl:8.1.0
+  schemaRegistryUrl: http://schema-registry-service:8081
+  config:
+    # - subject1:BACKWARD
```

Comment:
instead of splitting strings, you can use an array of objects. example,
```
config:
  - subject: topic1-values
    compatibility: BACKWARD
  - subject: topic2-values
    compatibility: FORWARD    
```

Reply:
That requires us to write the repeat for all subjects logic in helm templating instead of bash isn't it? Or am I missing something

Currently the approach will have both the information in one string and inside bash we split the data

Reply:
yes, we can use helm `range` to iterate over the list.